static/js/people.js

```diff
@@ -1,6 +1,6 @@
 import md5 from "blueimp-md5";
+import {formatToTimeZone} from "date-fns-timezone";
```

Comment:
`date-fns-timezone` looks unmaintained. The `date-fns` documentation [recommends](https://date-fns.org/docs/Time-Zones) `date-fns-tz`.